public/assets/css/styles.css

```diff
@@ -43,4 +43,12 @@ main {
 h1,
 h2 {
   @apply font-big;
-}
\ No newline at end of file
+}
+
+p {
+  @apply font-small text-2xl;
+}
+
+section {
+  @apply my-16;
```

Comment:
Use `my-12` and erase this class in section of `index`

Reply:
:heavy_check_mark: